src/app/dataset-block/metadata-block/components/event-details/components/set-polling-source-event/set-polling-source-event.component.html

```diff
@@ -0,0 +1,135 @@
+        </p>
+        <p>
+            &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;header:
+            <span class="value-color"> {{ readStepCsv.header }}</span>
+        </p>
+    </ng-container>
+    <p class="color-key">merge:</p>
+    <ng-container *ngIf="isMergeStrategyLedger">
+        <p>
+            &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;kind:
+            <span class="value-color"> {{ event.merge.__typename }}</span>
+        </p>
+        <p>
+            &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;primaryKey:
+            <span class="value-color">
+                {{ mergeStrategyLedger.primaryKey.join(", ") }}</span
+            >
+        </p>
+    </ng-container>
+
+    <ng-container *ngIf="event.preprocess">
+        <p class="color-key">preprocess:</p>
+        <p>
+            &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;kind:
+            <span class="value-color"> {{ event.preprocess.__typename }}</span>
+        </p>
+
+        <p>
+            &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;engine:
+            <span class="value-color"> {{ event.preprocess.engine }}</span>
+        </p>
+
+        <p *ngIf="event.preprocess.version">
+            &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;version:
+            <span class="value-color"> {{ event.preprocess.version }}</span>
+        </p>
+
+        <ng-container *ngIf="event.preprocess.queries">
+            <p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;queries:</p>
+            <ng-container *ngFor="let query of event.preprocess.queries">
+                <p>
+                    &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;- alias:<span
+                        class="value-color"
+                    >
+                        {{ query.alias ? query.alias : "none" }}</span
+                    >
+                </p>
+                <p>
+                    &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; query:<span
+                        class="value-color lh-sm"
+                    >
+                        {{ query.query }}</span
+                    >
+                </p>
+            </ng-container>
+        </ng-container>
+    </ng-container>
+</div>
+
+<ng-container *ngIf="!isYamlView">
+    <div class="d-flex flex-column gap-4 mt-4">
+        <div
+            class="position-relative border border-1 p-2 rounded-3"
+            *ngFor="let section of eventSections"
+        >
+            <span
+                class="position-absolute text-muted group-box-label text-uppercase"
+                >{{ section.title }}</span
+            >
+            <div class="mx-4">
+                <ng-container *ngFor="let row of section.rows">
+                    <app-block-row-data
+                        *ngIf="row.label !== queriesLabel"
+                        [tooltip]="row.tooltip"
+                        [label]="row.label"
+                    >
+                        <ng-template #container></ng-template>
+                    </app-block-row-data>
+                    <ng-container *ngIf="row.label === queriesLabel">
```

Comment:
Instead of detecting the case by text of the label, introduce an explicit boolean field in the row interface - something like `separate_row_for_value: bool`